### PR TITLE
Documenting debug:twig command

### PR DESCRIPTION
**Pull request by yceruto**
See https://github.com/symfony/symfony/pull/27981

### CODE REVIEW

**Comment by yceruto** [comment]
Related PR was merged :tada: now we can take care of this one :) Cheers!

File: templating/namespaced_paths.rst [hunk 1]
```diff
@@ -71,6 +71,13 @@ in the ``vendor/acme/foo-bar/templates/`` directory, you can refer to it as:
 
     {{ include('@foo_bar/sidebar.twig') }}
 
+Execute this command to verify if your template name is correct and know which
+template file will be loaded:
```
[the remark this answers is not in the document]
Comment (yceruto) [reply templating/namespaced_paths.rst]: Sadly, at this moment, the command only work for physical files.

File: templating/debug.rst [hunk 2]
```diff
@@ -67,3 +67,20 @@ By design, the ``dump()`` function is only available in the ``dev`` and ``test``
...
+
+.. code-block:: terminal
+
+    # list general information
+    $ php bin/console debug:twig
+
+    # filter output by any keyword
+    $ php bin/console debug:twig --filter=date
+
+Also, you can verify any template name to know which template file will be loaded:
```
[the remark this answers is not in the document]
Comment (yceruto) [reply templating/debug.rst]: Done, thanks!

File: templating/namespaced_paths.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (yceruto) [reply templating/namespaced_paths.rst]: Btw, I'm not sure if the `TwigBridge` & `TwigBundle` support such feature.

**Comment by javiereguiluz** [comment]
This was finally merged. Yonel, thanks as always for creating these great little utilities to improve developer experience!

**Pull request merged**